pdns/recursordist/rec-taskqueue.cc

```diff
@@ -181,13 +181,23 @@ void runTaskOnce(bool logErrors)
 
 void pushAlmostExpiredTask(const DNSName& qname, uint16_t qtype, time_t deadline)
 {
+  if (SyncRes::isUnsupported(qtype)) {
+    auto log = g_slog->withName("taskq")->withValues("name", Logging::Loggable(qname), "qtype", Logging::Loggable(QType(qtype).toString()));
```

Comment:
`Loggable<>::to_string()` template could use a `QType` specialization as it is a common type we may use in logging ?